integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml

```diff
@@ -85,7 +85,9 @@
   {{- $workerJavaOpts = print "-Dalluxio.worker.fuse.enabled=true" | append $workerJavaOpts }}
   {{- $workerJavaOpts = print "-Dalluxio.worker.fuse.mount.alluxio.path=/" | append $workerJavaOpts }}
   {{- $workerJavaOpts = printf "-Dalluxio.worker.fuse.mount.point=%v" .Values.fuse.mountPath | append $workerJavaOpts }}
-  {{- $workerJavaOpts = print "-Dalluxio.worker.fuse.mount.options=allow_other" | append $workerJavaOpts }}
+  {{- range .Values.fuse.mountOptions}}
```

Comment:
I do not quite understand why we use .Values.fuse.mountOptions like that

Reply:
I would prefer to use one line for all fuse mount options like allow_other,kernel_cache,attr_timeout=9000.
and use
```
{{- $workerJavaOpts = print "-Dalluxio.worker.fuse.mount.options=.Values.fuse.mountOptions" | append $workerJavaOpts }}
```
here

Reply:
I modified it to be how jvmOptions work. The options would look clearer as an array on multiple lines in values.yaml, just like jvmOptions. Then using concat instead of loop and append can make them all on one line. Do you think this should be the way to go?